firejail.nimble

```diff
@@ -10,3 +10,6 @@ srcDir        = "src"
 # Dependencies
 
 requires "nim >= 0.19.4"
+
+import distros
+foreignDep "firejail"
```

Comment:
Maybe `webp` will need something similar `foreignDep "libwebp"` ? :thinking:

Reply:
Yes, I do think so! Try to uninstall libwebp and `nimble install webp` ;)